app/helpers/application_helper/button/chargeback_rate_remove.rb

```diff
@@ -2,7 +2,7 @@ class ApplicationHelper::Button::ChargebackRateRemove < ApplicationHelper::Butto
   needs :@record
 
   def disabled?
-    if @record.default?
+    if @record.default? || @record.description == 'Default Container Image Rate'
```

Comment:
@zeari I think this should be in a constant and then references in all the places. I'll merge now and you can do it in a followup PR.